base/log.go

```diff
@@ -44,12 +44,12 @@ func DatasetLog(ctx context.Context, r repo.Repo, ref reporef.DatasetRef, limit,
 						if ds, err := dsfs.LoadDataset(ctx, r.Store(), v.Path); err == nil {
 							if ds.Commit != nil {
 								items[i].CommitMessage = ds.Commit.Message
-								items[i].CommitTitle = ds.Commit.Title
 							}
 						}
 					}
 					versions[i].Foreign = !local
 					items[i].VersionInfo = versions[i]
+					items[i].CommitTitle = notes[i]
```

Comment:
For better fill rate of the `DatasetLogItem` we now rely on the `op.Log` to provide the title so this works well for foreign datasets as well.

Reply:
Also need to update the above comments as they are incorrect.